packages/react/src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -3,14 +3,31 @@ import { screenReaderContainerStyles } from '../../../../utils/accessibility/Sty
 import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles'
 import { default as ListItem, ListItemProps } from '../../../../components/List/ListItem'
 import getBorderFocusStyles from '../../getBorderFocusStyles'
+import { ListItemVariables } from './listItemVariables'
+
+export type ListItemStylesProps = Pick<
+  ListItemProps,
+  | 'debug'
+  | 'important'
+  | 'navigable'
+  | 'selectable'
+  | 'selected'
+  | 'truncateContent'
+  | 'truncateHeader'
+> & {
+  hasContent: boolean
```

Comment:
These should be optional I guess?

Reply:
![image](https://user-images.githubusercontent.com/14183168/73185413-4d4df380-411e-11ea-9a30-48d61da7166b.png)

According to component's definition, they will be defined always

Reply:
It just feels odd all other to be optional, but these required, at least from the APIs point of view of the `ListStylesProps`

Reply:
Updated 👍